BetaCameras/BaslerACA/BaslerACA.cs

```diff
@@ -0,0 +1,147 @@
+﻿// Copyright (c) Metrilus GmbH
+// MetriCam 2 is licensed under the MIT license. See License.txt for full license text.
+
+using Metrilus.Util;
+using System;
+using System.Linq;
+using System.Drawing;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using PylonC.NETSupportLibrary;
+using PylonC.NET;
+
+namespace MetriCam2.Cameras
+{
+    public class BaslerACA : Camera
```

Comment:
The Vendor property should be overwritten, the modelName set (in this case I'd set it to "Ace" in the c'tor, and then update it during Connect. Maybe also update it when DeviceClass (see comment below) is updated.